tests.sh

```diff
@@ -119,6 +119,11 @@ build_csharp() {
   (cd csharp/src; dotnet restore)
   csharp/buildall.sh
   cd conformance && make test_csharp && cd ..
+
+  # Run csharp compatibility test.
+  cd csharp/compatibility_tests/v3.0.0
```

Comment:
Perhaps it would be cleaner if the test script used its own location automatically? Then we could just run

`csharp/compatibility_tests/v3.0.0/test.sh`

which would be simpler.